src/vcpkg/commands.integrate.cpp

```diff
@@ -459,12 +459,12 @@ CMake projects should use: "-DCMAKE_TOOLCHAIN_FILE=%s"
                             .string_arg(buildsystems_dir)
                             .string_arg(nuspec_file_path);
 
-        const int exit_code =
-            cmd_execute_and_capture_output(cmd_line, default_working_directory, get_clean_environment()).exit_code;
+        const auto result =
```

Comment:
Since you may move from this now should we drop const?

Reply:
Later in the function `Checks::exit_success(VCPKG_LINE_INFO);` is called, so you probably don't move from this.